### ISSUE TITLE
Cacti 1.1.7 + NWM latest git

### ISSUE THREAD

**Issue by jpobeda** [howardjones/network-weathermap#104]
2 different issues

1.- When I go to Management / Weathermap sometimes the warnings on every map don't load the exclamation image correctly. Look at the screenshot attached (nwm_#104_1). Sometime it loads it sometimes it doesn't.

File permission look ok. Browser Issue?

> ls -al images/exclamation.png
> -rw-r--r--. 1 cacti apache 701 May 22 11:49 images/exclamation.png
>  ls -Z images/exclamation.png
> -rw-r--r--. cacti apache system_u:object_r:usr_t:s0       images/exclamation.png

2.- When I click on any Warning map icon to get to CLOG filtered by the map name, It doesn't apply it. The screenshot (nwm_#104_2) would explain it better I think.

![nwm_ 104_2](https://cloud.githubusercontent.com/assets/13456559/26605805/8c68aefe-45e3-11e7-8c29-4175cb65bc23.png)

![nwm_ 104_1](https://cloud.githubusercontent.com/assets/13456559/26605843/b789a3a4-45e3-11e7-89f7-1562e153207a.png)
)

**Comment by howardjones** [comment]
There are a number of issues with the 1.x plugin that I don't understand, and don't have enough information to faultfind easily. I've been putting these off because of that.  I noticed the log one too. I guess the URL for filtering the logs has changed. In another situation, sometimes the page comes up with no styling at all, and sometimes it doesn't.

**Comment by jpobeda** [comment]
Yes, you are right, I've noticed the same but I thought it was something with my browser because after an F5 it comes ok so...

**Comment by jpobeda** [comment]
Btw, I updated my local copy of weathermap-cacti10-plugin-mgmt.php to solve the clog issue.

Was before:
` print '<br><a href="../../utilities.php?tail_lines=500&message_type=2&action=view_logfile&filter='.urlencode($map->configfile).'" `

Now:
`print '<br><a href="../../utilities.php?tail_lines=500&message_type=2&refresh=60&reverse=1&rfilter='.urlencode($map->configfile).'&filename=cacti.log&action=view_logfile&" `

**Comment by netniV** [comment]
Is this resolved now?

**Comment by howardjones** [comment]
No - the new UI will need to have this (and 0.8.8 and 1.0.0 variations too)
